packages/waku/src/lib/builder/build.ts

```diff
@@ -495,9 +495,11 @@ const createTaskRunner = (limit: number) => {
 const WRITE_FILE_BATCH_SIZE = 2500;
 const { runTask, waitForTasks } = createTaskRunner(WRITE_FILE_BATCH_SIZE);
 
-const emitStaticFile = (
+export { waitForTasks };
+
+export const emitStaticFile = (
```

Comment:
Make it something like,

```suggestion
// This is exported for vite-rsc. https://github.com/wakujs/waku/pull/1493
export { waitForTasks };

// This is exported for vite-rsc. https://github.com/wakujs/waku/pull/1493
export const emitStaticFile = (
```

as well as other similar changes and open a separate PR please.